.env

```diff
@@ -0,0 +1,9 @@
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=123456
+POSTGRES_DATABASE=book_service
+POSTGRES_LOCAL_PORT=5434
+POSTGRES_DOCKER_PORT=5432
+
+SPRING_LOCAL_PORT=8088
+SPRING_DOCKER_PORT=8080
+DEBUG_PORT=5005

```

Comment:
add .env.template to the project with actual values removed, add .env to gitignore, use git rm --cached .env to keep local copy but remove from remote origin